source/git.js

```diff
@@ -292,58 +305,104 @@ git.discardChangesInFile = function(repoPath, filename) {
   return task;
 }
 
+var parseDiffForPatch = function (patch, repoPath) {
+  return new Promise(function (resolve, reject) {
+    git(['diff', patch.name], repoPath)
+      .fail(reject)
+      .done(resolve).start();
+  });
+}
+
+var applyPatchedDiff = function(patch, repoPath, patchedDiff) {
+  return new Promise(function (resolve, reject) {
+    if (patchedDiff) {
+      git(['apply', '--cached'], repoPath)
+        .fail(reject)
+        .done(resolve)
+        .started(function() {
+          this.process.stdin.end(patchedDiff + '\n');
```

Comment:
Adding an additional `\n` resolved https://github.com/FredrikNoren/ungit/pull/602#issuecomment-124866925 for me on windows.

Complete line:
```js
this.process.stdin.end(patchedDiff + '\n\n');
```
Can you check if this modification does also work on your os?